examples/configs/devices/Eyelink_1.yml

```diff
@@ -0,0 +1,11 @@
+device_id: Eyelink_1
+device_sn: 192.168.100.15
+device_name: EYELINK Portable Duo
+device_location: '25_19_36'
+wearable_bool: false
+device_make: SR
+device_model: 'EYELINK CL 6.50  '
```

Comment:
Why the trailing spaces?

Reply:
This was an automated transfer of data from the db, so the trailing spaces originated there. I can remove